script/Arcueid/Day 6/06_00_ARC06_2_1.txt

```diff
@@ -146,8 +146,9 @@ After all, I've been able to put up with it just fine so far."
 [sha:bb8f59c90e3982c3e1ec332650abdafc5217e3e9]{
 -- Page 186, Offset 14361.
 -- 「……はあ。秋葉さまのお気持ちがちょっとだけ分かっちゃいました。志貴さんは平気なお顔をしていても、いえ、している時の方がたいへんな方なんですね。
-// Don't love using "sigh" for an actual sigh
-"...Sigh. I think I'm starting to understand how Akiha-sama feels. Yet despite it all, your face looks so calm... no, rather, it's when you look this calm that you must be suffering the most, right?
+"...Sigh. I think I'm starting to understand how Akiha-sama feels.
+ Yet despite your face looking so calm...
+ No, rather, it's when you look this calm that you must be having it the worst, right?
```

Comment:
'must be having it the worst,' is a weird way to put that, maybe 'must have it the worst'. Although for this section I actually like the original 'must be suffering the most' best.

Reply:
Yeah, I remain partial to suffering as well. Dunno if either of 
> must be hurting the most

or 
> must be struggling the most 

appeal to anti-sufferers?

Reply:
@Avuxo @yk1512 thoughts?

Reply:
I do agree that 'suffering' might be a tad strong but couldn't think of a better alternative. I'm for keeping it.

Reply:
I'm not one to fight the wind's heading, it can stay I suppose.